docs/nix.md

```diff
@@ -107,6 +107,37 @@ dxu@kashmir bpftrace]$ CC=afl-clang-fast CXX=afl-clang-fast++ cmake -B build-fuz
 
 This section has a few examples on how to interact with the Nix configuration.
 
+### Update flake inputs
+
+Flakes have external inputs in the `inputs = { ... }` section of `flake.nix`.
+
+To update a single input:
+```
+$ nix flake lock --update-input blazesym
```

Comment:
This warns me with:
```
> warning: '--update-input' is a deprecated alias for 'flake update' and will be removed in a future version.
```

Reply:
Interesting. Can't seem to find an equivalent, at least not in the help page. Tried a few things:

```
dxu@kashmir~/d/bpftrace (blazesym_update) $ nix flake update blazesym
error: cannot find flake 'flake:blazesym' in the flake registries
dxu@kashmir~/d/bpftrace (blazesym_update) $ nix flake update --update-input blazesym
error: unrecognised flag '--update-input'
Try 'nix --help' for more information.
dxu@kashmir~/d/bpftrace (blazesym_update) $ nix flake update --input blazesym
error: unrecognised flag '--input'

```

Reply:
Weird. I suppose you may have to update `nix` itself?

For me the following works (in the sense that it doesn't error out), but produces the same warning:
```
$ nix flake update --update-input blazesym
warning: '--update-input' is a deprecated alias for 'flake update' and will be removed in a future version.
```
...
Anyway, no idea.

Reply:
Yeah, I've got nix 2.17 which is a bit old. Since it looks like a new thing, maybe let's leave it as is for now, so it works for the most people.